### PR TITLE
[bugfix] fix numpy sanity check command + cleaner output under -x

### PR DESCRIPTION
**Pull request by boegel**
We need to use the full path to `multiarray.so` to avoid problems with the sanity check.

### CODE REVIEW

File: easybuild/easyblocks/n/numpy.py [hunk 1]
```diff
@@ -228,7 +228,10 @@ def test_step(self):
             res = sec_re.search(out)
             if res:
                 time_msec = 1000 * float(res.group('time'))
-            elif not self.dry_run:
+            elif self.dry_run:
+                # use fake value during dry run
```
Comment (wpoely86) [inline easybuild/easyblocks/n/numpy.py]: warning about fake value?

Comment (boegel) [inline easybuild/easyblocks/n/numpy.py]: sort of useless under dry run, but sure, why not

File: easybuild/easyblocks/n/numpy.py [hunk 2]
```diff
@@ -256,8 +259,8 @@ def sanity_check_step(self, *args, **kwargs):
         ]
         if LooseVersion(self.version) >= LooseVersion("1.10"):
             # if blas_dot symbol is not there, numpy isn't properly linked against a BLAS library
-            multiarray_so = os.path.join(self.pylibdir, 'numpy', 'core', 'multiarray.so')
-            custom_commands.append("nm %s | grep blas_dot" % multiarray_so)
+            multiarray_so = os.path.join(self.installdir, self.pylibdir, 'numpy', 'core', 'multiarray.so')
+            custom_commands.append("nm %s | grep blas_dot" % self.installdir(multiarray_so)
```
Comment (wpoely86) [inline easybuild/easyblocks/n/numpy.py]: why do you still need `installdir` here?

Comment (boegel) [inline easybuild/easyblocks/n/numpy.py]: I was testing you. ;-)

No, typo, this is actually a syntax error...

Comment (wpoely86) [inline easybuild/easyblocks/n/numpy.py]: I was wondering what this was supposed to do. Use `synaptics` or such?

Comment (boegel) [inline easybuild/easyblocks/n/numpy.py]: I have PyLint set up, but wasn't paying close enough attention, my apologies. I was going to test this before merging it though. :-)

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**